demo/form/input/playground

```diff
@@ -1,3 +1,168 @@
+class SimpleLoginForm extends React.Component {
```

Comment:
this is no simple login form, do we really need this here? I think this form shouldn't be part of this playground but in fieldset's playground

Reply:
To understand how to use this component i think a real use case in a real form like this is needed. In the `Fieldset` we can have another form to show the integration of that component in a form.
Also, it shows how to implement some uses of radio buttons like the example with the images as buttons to select a "category"
Besides this, the name of the class it's not right so i'll change it to reflect the purpose of the form